examples/pbfhr/multi_pt/prt_2ref/input_10s.py

```diff
@@ -1,206 +0,0 @@
-''' changed from the initial input file
```

Comment:
I didn't delete them because they fails, but because they were redundant. Now all input files run, including those in sfr and default folders. I didn't wait to see the final results through. The input files in sfr didn't work because of the convective heat transfer calculation in dtempdt(th_system.py), I corrected it.